Orange/widgets/data/tests/test_owselectrows.py

```diff
@@ -310,37 +310,6 @@ def test_partial_match_values(self):
         self.assertEqual(condition[1], 2)
         self.assertEqual(condition[2], (2, ))  # index of value + 1
 
-    def test_backward_compat_match_values(self):
```

Comment:
If I understand correctly from `# After 2022/2/2, use just the expression in else clause` indices instead of values in context values also not supported anymore. Removing this test.